api/envoy/api/v2/route/route.proto

```diff
@@ -104,6 +104,28 @@ message VirtualHost {
   // specific; see the :ref:`HTTP filter documentation <config_http_filters>`
   // for if and how it is utilized.
   map<string, google.protobuf.Struct> per_filter_config = 12;
+
+  message RetryPriority {
+    string name = 1;
+    google.protobuf.Struct config = 2;
+  }
+
+  // Specifies an implementation of a RetryPriority which is used to determine the
+  // priority load used for retries.
```

Comment:
Priority load seems vague to me. Maybe "distribution of load across priorities used for retries."